CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 - [#5447](https://github.com/thanos-io/thanos/pull/5447) Promclient: Ignore 405 status codes for Prometheus buildVersion requests
 - [#5451](https://github.com/thanos-io/thanos/pull/5451) Azure: Reduce memory usage by not buffering file downloads entirely in memory.
+- [#5487](https://github.com/thanos-io/thanos/pull/5487) Project fully built with Go 1.18 (including Github Actions).
```

Comment:
It's not relevant for end users, but does this mean it doesn't deserve to go into the changelog? 🤔 

i.e. end users might be happy to know that we're building with 1.18 because of improved performance in arm64.

Reply:
So in the end, I believe the initial PR that added the build with Go 1.18 deserves to be in the CHANGELOG. In this one I am just updating some Github Actions, so I could add #5258 to the changelog and remove this one.

Reply:
Removed the changelog entry for this PR in the change (see https://github.com/thanos-io/thanos/pull/5487/commits/55a26f3c25e1505fbd41cb04479288f7b728ceb2)

Reply:
Thanks for the explanation 👍